### DISCUSSION TITLE
Can you format description when using Nordigen (or Spectre)

### DISCUSSION THREAD

**Discussion by sanderisbestok** [firefly-iii/firefly-iii#6125] [Firefly III] [answered]
I just configured Firefly and it works quite good. But when importing through Nordigen, the format my bank (ING) gives, is just messed up. The description contains a lot more information and actual HTML tags. Is there a way to format the description? Or is it better to use Spectre (they also have an option for ING). 

I have provided a screenshot where you can see the mess coming from the API, is there something I can do about it?

![Screenshot from 2022-06-01 18-24-24](https://user-images.githubusercontent.com/15033290/171454826-89d81fb5-60f0-46ad-b7bc-aa2a659592b1.png)

**Comment by JC5** [comment, marked as answer]
You can use rules to trigger on individual entries in the description. The data importer can't format this for you yet.
